src/System.Console/tests/ConsoleEncoding.cs

```diff
@@ -11,6 +11,8 @@
 
 public partial class ConsoleEncoding : RemoteExecutorTestBase
 {
+    public static bool ConsoleEncodingAvailable => !PlatformDetection.IsUap || PlatformDetection.IsWindows10RS3OrGreater;
```

Comment:
Not sure what you mean by list of tests that were run on the CI. Everything not outerloop or disabled is run on the CI, on the configurations listed...

I guess I'm asking why you have an RS3 conditoin -- do you know it fails on RS2?